static/css/bureau-structure.less

```diff
@@ -87,14 +92,14 @@
         });
 
         &_name {
-            .respond-to-min(600, {
+            .respond-to-min(600px, {
                 .h4();
                 margin-bottom: unit(15px / @font-size, em);
```

Comment:
Check that `@font-size` shouldn't be `@base-font-size-px`. Looks like this should be re-written as:

```
unit( (@grid_gutter-width / 2) / @base-font-size-px, em);
```